qualtran/bloqs/util_bloqs.py

```diff
@@ -166,9 +165,7 @@ def add_my_tensors(
     def on_classical_vals(self, reg: 'NDArray[np.uint8]') -> Dict[str, int]:
         return {'reg': bits_to_ints(reg)[0]}
 
-    def get_ctrl_system(
-        self, ctrl_spec: Optional['AbstractCtrlSpec'] = None
-    ) -> Tuple['Bloq', 'AddControlledT']:
+    def get_ctrl_system(self, ctrl_spec=None) -> Tuple['Bloq', 'AddControlledT']:
```

Comment:
what was wrong with the annotation that used to be here?

Reply:
AbstractCtrlSpec doesn't exist?  Was this refactored?  I couldn't find it anywhere.  (Unless I missed it)

Reply:
OH -- yeah that would be a problem. I squashed `CtrlSpec` and `AbstractCtrlSpec` into `CtrlSpec` during that PR but must have missed this

Reply:
Changed to Optional['CtrlSpec']